packages/wallets/src/components/WalletTourGuide/DesktopSteps.tsx

```diff
@@ -7,8 +7,7 @@ const getDesktopSteps = (
     isDemoWallet: boolean,
```

Comment:
For now we are unclear when the revamped (video) onboarding is being released as it is under discussion that it might cause performance issues so I believe we should keep the current onboarding updated and not broken before the user release